extensions/binding/org.eclipse.smarthome.binding.astro/OSGI-INF/AstroDiscoveryService.xml

```diff
@@ -9,7 +9,7 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.astro.discovery">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="false" name="org.eclipse.smarthome.binding.astro.discovery">
```

Comment:
That's because it is providing a service instance and as soon as the consumer(s) of this service request(s) this service, this component will be active.